lambda/step-lambda.js

```diff
@@ -36,10 +36,13 @@ exports.handler = async function(event, context) {
     ReturnValues:"UPDATED_NEW"
   }
   console.log("Calling DDB", JSON.stringify(request))
-  const result = await documentClient.update(request).promise()
-  
-  //console.log("Context", JSON.stringify(context));
-  console.log(JSON.stringify(result));
 
-  return {"statusCode": 200, "body": JSON.stringify(result)}
+  try {
+    const result = await documentClient.update(request).promise()
+    return {"statusCode": 200, "body": JSON.stringify(result)}
+  }
+  catch(error){
+    return {"statusCode": error.statusCode, "body": error.message}
```

Comment:
`error` here is unlikely to have a `statusCode` field so this might blow up. Similarly, `error` might not have a `message` field depending on what is thrown (anything can be thrown, not just exceptions). I'd be super cautious here and log `error` and hardcode the `statusCode` to `503` or similar.